Sources/StreamChat/Utils/Debouncer.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol Debouncing {
```

Comment:
Where is the debouncer going to be used? If it's not in the LLC, then we should probably move it to the UI layer, since it's not public (which is correct, we shouldn't expose this).

Reply:
My understanding is that the AudioPlayer will also be moved to the LLC. The PR is just not updated it yet

Reply:
And do we actually need a debouncer for this? https://github.com/GetStream/stream-chat-swift/pull/2529#discussion_r1139203252

AFAIK AVFoundation is already quite forgiving if we do multiple seek requests, it should handle it automatically